modules/ROOT/pages/project-best-practicesIndex.adoc

```diff
@@ -0,0 +1,26 @@
+= Project best practices
+:description: blabla
+
+blabla
+[.card-section]
+== A few reminders we found useful to recall here
+
+[.card.card-index]
+--
+xref:lifecycle-and-profiles.adoc[[.card-title]#Project phases# [.card-body.card-content-overflow]#pass:q[]#]
+--
+
+[.card.card-index]
+--
+xref:design-methodology.adoc[[.card-title]#Conception# [.card-body.card-content-overflow]#pass:q[TODO]#]
+--
+
+[.card.card-index]
+--
+xref:setupDevEnvironmentIndex.adoc[[.card-title]#Set up the development environment# [.card-body.card-content-overflow]#pass:q[]#]
```

Comment:
I don't think that the set up documentation is really related with project best practices. By that I mean that I would never look into the best practices if i was looking for a guide to setup my environment. I would probably expect it to be in the quick start.

Reply:
ok thx